fml/thread_unittests.cc

```diff
@@ -63,10 +109,10 @@ static void MockThreadConfigSetter(const fml::Thread::ThreadConfig& config) {
   int policy = SCHED_OTHER;
   switch (config.priority) {
     case fml::Thread::ThreadPriority::kDisplay:
-      param.sched_priority = 10;
+      param.sched_priority = clamp_priority(10, policy);
```

Comment:
Should the value perhaps be `sched_get_priority_min` and `sched_get_priority_max`? The values are platform specific (i.e. on macOS the allowed range is 15-47 for SCHED_OTHER, on Linux it is 0-0 for SCHED_OTHER and for other policies it requires root).

I don't quite understand how these tests were passing at all on linux.

Reply:
Ouch, actually, I know how. The tests were not ran at all - because of missing `build_config.h` include `FLUTTER_PTHREAD_SUPPORTED` was 0 all the time.